housekeeping-vacuum-tool/README.md

```diff
@@ -74,11 +75,20 @@ The Vacuum tool already has the required H2 drivers on its CLASSPATH so the only
       schema-name: my_db
       h2:
           # Location of H2 DB on filesystem
-          database: /home/hadoop/vacuumtest/mydb/housekeeping
+          database: /home/hadoop/vacuumtest/mydb/${housekeeping.schema-name}
       dataSource:
           username: user
           password: foo
 
+If the schema does not already exist, a `db-init-script` can be provided to initialise it, as shown in the following example:
```

Comment:
I think this example should be moved to the housekeeping README if possible, this holds for everything that uses housekeeping not just VT